### PR TITLE
chore: doc updates

### PR DESCRIPTION
**Pull request by peri044**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

File: docsrc/index.rst [hunk 1]
```diff
@@ -54,6 +54,11 @@ User Guide
    tutorials/_rendered_examples/dynamo/vgg16_ptq
    tutorials/_rendered_examples/dynamo/engine_caching_example
    tutorials/_rendered_examples/dynamo/refit_engine_example
+   tutorials/serving_torch_tensorrt_with_triton
```
Comment (narendasan) [inline docsrc/index.rst:57]: I still want a tutorials section for stuff that is in beta or outside the mainline use cases

File: docsrc/index.rst [hunk 2]
```diff
@@ -99,30 +104,27 @@ FX Frontend
 
    fx/getting_started_with_fx_path
 
-Tutorials
+Model Zoo
```
Comment (narendasan) [inline docsrc/index.rst:107]: You can still separate Model Zoo and tutorials

**Comment by narendasan** [comment]
Did you fix the heading issues with GPT and LLaMA as well? 
image

**Review by lanluo-nvidia — APPROVED** [review]
LGTM

**Pull request merged**